src/settings/_font.scss

```diff
@@ -6,29 +6,35 @@ $ff-sans-serif: 'Open Sans', Helvetica, sans-serif !default;
 // Font weights
 $fw-light: 300 !default;
 $fw-regular: 400 !default;
-$fw-semi-bold: 600 !default;
+$fw-semi-bold: 500 !default;
 $fw-bold: 700 !default;
 
 // Font sizes
-$fz-base: 14 !default;
-$fz-body: $fz-base * 1px !default;
-$fz-xs: ceil($fz-base * .666) * 1px !default;
-$fz-s: ceil($fz-base * .8) * 1px !default;
-$fz-m: $fz-base * 1px !default;
-$fz-l: ceil($fz-base * 1.25) * 1px !default;
-$fz-xl: ceil($fz-base * 1.75) * 1px !default;
+$fz-m: 16px !default;           // 16px
+$fz-giant: $fz-m + 20 !default; // 36px
+$fz-xxxl: $fz-m + 12 !default;  // 28px
+$fz-xxl: $fz-m + 8 !default;    // 24px
+$fz-xl: $fz-m + 4 !default;     // 20px
+$fz-l: $fz-m + 2 !default;      // 18px
+$fz-s: $fz-m - 2 !default;      // 14px
+$fz-xs: $fz-m - 4 !default;     // 12px
+$fz-xxs: $fz-m - 6 !default;    // 10px
 
-// Font sizes for headings
-$fz-h1: 32px !default;
-$fz-h2: 24px !default;
-$fz-h3: 18px !default;
-$fz-h4: 14px !default;
-$fz-h5: 12px !default;
+// Semantic font sizes
+$fz-h1: $fz-giant !default; // 36px
```

Comment:
I think you might have used the margins and not the font sizes
![basic_styles_-_components_ui_-_zeplin](https://user-images.githubusercontent.com/17549662/30922945-ec0b45ea-a3aa-11e7-8dd4-bb8190cdf449.png)

Reply:
Definitive values should be relatives the one to the other. Not absolute random values. But if our UX pals are agree I won't be arguing 'bout that.